### PR TITLE
OPS#3923 - Fix NSF status for online banking

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov-registries/ops-support/issues/3923

Fix nsf status for online banking

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2820-5gjryc15.web.app

File: auth-web/package-lock.json [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+  "version": "2.6.11",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "auth-web",
-      "version": "2.6.10",
+      "version": "2.6.11",
       "dependencies": {
-        "@bcrs-shared-components/base-address": "2.0.3",
```
Comment (seeker25) [inline auth-web/package-lock.json:11]: yikes let me fix this?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2820-5gjryc15.web.app

**Pull request merged**